src/pgduckdb_types.cpp

```diff
@@ -962,7 +971,7 @@ struct PostgresArrayAppendState {
 	idx_t count = 0;
 
 public:
-	idx_t expected_values = 1;
+	uint64 expected_values = 1;
```

Comment:
```suggestion
	uint64_t expected_values = 1;
```

Reply:
IIRC uint64 != uint64_t (at least on Mac) so it creates compiler errors. I'll check asap

Reply:
Weird

Reply:
Ugh, let's leave it as is then...

Reply:
This is changed in PG18 btw, where the Postgres types will simply be aliases for the `_t` variants: https://github.com/postgres/postgres/commit/962da900ac8f0927f1af2fd811ca67fa163c873a

Reply:
> Ugh, let's leave it as is then...

You mean keep it an `idx_t`? That won't work either since `idx_t` is also an `uint64_t` :-)

Reply:
So `uint64`